build.sbt

```diff
@@ -66,8 +66,44 @@ lazy val root = (project in file("."))
       circeGeneric,
       circeGenericExtras,
       circeParser,
-      ujson
+      ujsonLib
     ),
+    // Source generator: produce only the BaseSchema object with property name constants
+    Compile / sourceGenerators += Def.task {
```

Comment:
Is it possible to abstract this out of the main `build.sbt` either into a "sub" sbt or a scala class that can then be called in the main `build.sbt` to make the it cleaner?